src/ngraph/runtime/cpu/builder/quantized_conv.cpp

```diff
@@ -106,20 +106,61 @@ namespace ngraph
                     auto& out0_tensor = external_function->get_tensor_data(out[0].get_name());
 
                     auto& mkldnn_emitter = external_function->get_mkldnn_emitter();
+                    bool is_bias_i32 = (args[2].get_element_type() == element::i32);
+                    if (!is_bias_i32)
+                    {
+                        auto reorder_bias_desc = mkldnn_emitter->build_memory_descriptor(
+                            args[2].get_shape(), ngraph::element::i32, mkldnn::memory::format::x);
+                        auto conv_index =
+                            mkldnn_emitter->build_bias_reorder(node, args, out, reorder_bias_desc);
```

Comment:
Please modify build_convolution<>() for this instead of duplicating code in another function. You can do the args[2].get_element_type() check inside build_convolution as well, create reorder descriptors there.

Reply:
you dont need to return a vector. See this: https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/runtime/cpu/builder/max_pool.cpp#L129